docs/build.md

```diff
@@ -99,8 +99,7 @@ metadata:
     build.shipwright.io/verify.repository: "false"
 spec:
   source:
-    url: https://github.com/qu1queee/taxi
-    revision: master
+    url: https://github.com/shipwright-io/sample-go
```

Comment:
Just for correctness: should we have a `contextDir` here ?

Reply:
no needed, we default to the `default` branch of this repo, which is `main`

Reply:
That part I get. I mean that the `sample-go` repository has nothing in its root and we should therefore specify `contextDir: docker-build`?

Reply:
wirklich, fixing in a next push